databrowser/src/components/language/LanguagePicker.vue

```diff
@@ -3,7 +3,7 @@
     class="inline-flex md:hidden items-center"
     @click="showMobileSelect = true"
   >
-    <span class="pr-2">{{ currentSelected }}</span>
+    <span class="pr-2">{{ currentSelected.toUpperCase() }}</span>
```

Comment:
When a user reads this with a screen reader then the screen reader reads him for example "DE". I would add here an alternative description text for it. You can take a look at the Bookiply lanugage picker on desktop.